bot/models/migrations/006_up__configs.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS configs
+(
+    id                          BIGINT PRIMARY KEY DEFAULT create_snowflake(),
```

Comment:
I recant my previous statement and now agreed that we don't need a id column on all tables.
The primary key here should be guild_id